amoro-ams/src/main/resources/mysql/ams-mysql-init.sql

```diff
@@ -112,28 +112,27 @@ CREATE TABLE `table_metadata`
 CREATE TABLE `table_runtime`
 (
     `table_id`                      bigint(20) NOT NULL,
-    `catalog_name`                  varchar(64) NOT NULL COMMENT 'Catalog name',
-    `db_name`                       varchar(128) NOT NULL COMMENT 'Database name',
-    `table_name`                    varchar(256) NOT NULL COMMENT 'Table name',
-    `current_snapshot_id`           bigint(20) NOT NULL DEFAULT '-1' COMMENT 'Base table current snapshot id',
-    `current_change_snapshotId`     bigint(20) DEFAULT NULL COMMENT 'Change table current snapshot id',
-    `last_optimized_snapshotId`     bigint(20) NOT NULL DEFAULT '-1' COMMENT 'last optimized snapshot id',
-    `last_optimized_change_snapshotId`     bigint(20) NOT NULL DEFAULT '-1' COMMENT 'last optimized change snapshot id',
-    `last_major_optimizing_time`    timestamp NULL DEFAULT NULL COMMENT 'Latest Major Optimize time for all partitions',
-    `last_minor_optimizing_time`    timestamp NULL DEFAULT NULL COMMENT 'Latest Minor Optimize time for all partitions',
-    `last_full_optimizing_time`     timestamp NULL DEFAULT NULL COMMENT 'Latest Full Optimize time for all partitions',
-    `optimizing_status_code`        int DEFAULT 700 COMMENT 'Table optimize status code: 100(FULL_OPTIMIZING), 200(MAJOR_OPTIMIZING), 300(MINOR_OPTIMIZING), 400(COMMITTING), 500(PLANING), 600(PENDING), 700(IDLE)',
-    `optimizing_status_start_time`  timestamp(3) default CURRENT_TIMESTAMP(3) COMMENT 'Table optimize status start time',
-    `optimizing_process_id`         bigint(20) NOT NULL COMMENT 'optimizing_procedure UUID',
-    `optimizer_group`               varchar(64) NOT NULL,
-    `table_config`                  mediumtext,
-    `optimizing_config`             mediumtext,
-    `pending_input`                 mediumtext,
-    `table_summary`                 mediumtext,
+    `group_name`                    varchar(64) NOT NULL,
+    `status_code`                   int DEFAULT 0 NOT NULL COMMENT 'Table runtime status code.',
```

Comment:
0 status_code makes no sense, do you mean 700(IDLE)?

Reply:
OK.